aries/aries_vcx_core/src/wallet2/indy_wallet/indy_did_wallet.rs

```diff
@@ -0,0 +1,145 @@
+use async_trait::async_trait;
+use vdrtools::{DidMethod, DidValue, KeyInfo, Locator, MyDidInfo};
+
+use crate::{
+    errors::error::{AriesVcxCoreError, VcxCoreResult},
+    wallet2::{DidData, DidWallet},
+};
+
+#[async_trait]
+impl DidWallet for crate::wallet::indy::IndySdkWallet {
+    async fn create_and_store_my_did(
+        &self,
+        seed: &str,
```

Comment:
The existing `BaseWallet` interface has this as `seed: Option<&str>`. When `None` is specified, a random seed is generated. In fact, most of the code where `create_and_store_my_did` is being called passed `None`. 
Is there a reason why, seemingly, you've decided to strip this feature?

Reply:
I reverted it back, it is `Option<&str>` now.

Reply:
I see it as a simpler and safer API design force the client to explicitly choose the RNG and randomness source instead of allowing multiple code paths by hiding an implementation which may unknowing change beneath their feet.